### PR TITLE
Adding Broker CLI tool

### PR DESCRIPTION
**Pull request by deepika-chander**

### CODE REVIEW

File: broker-cli/LICENSE [hunk 1]
```diff
@@ -0,0 +1,202 @@
+
```
Comment (martinmaly) [inline broker-cli/LICENSE]: This file is already present at the root so we don't need to duplicate it.

File: .idea/codeStyles/codeStyleConfig.xml [hunk 2]
```diff
@@ -0,0 +1,5 @@
+<component name="ProjectCodeStyleConfiguration">
```
Comment (martinmaly) [inline .idea/codeStyles/codeStyleConfig.xml]: please delete the .idea directory from the PR

**Comment by deepika-chander** [comment]
Done.

File: .idea/codeStyles/codeStyleConfig.xml (hunk 2 above)
Comment (deepika-chander) [reply .idea/codeStyles/codeStyleConfig.xml]: done

File: broker-cli/LICENSE (hunk 1 above)
Comment (deepika-chander) [reply broker-cli/LICENSE]: done

File: broker-cli/cmd/bindings.go [hunk 3]
```diff
@@ -0,0 +1,293 @@
...
+		AcceptsIncomplete: true,
+		APIVersion:        apiVersion,
+	})
+	if err != nil {
+		return err
+	}
+
+	op, err := waitOnOperation(pollBindingOpFunc(client, flags.ApiVersionDefault, brokerURL, i.ID, b.ID, i.ServiceID, i.PlanID, res.OperationID, adapter.OperationDelete), showProgress)
+	if err != nil {
+		return fmt.Errorf("Error polling last operation %q for binding to instance %q in broker %q: %v", res.OperationID, b.ID, i.ID, brokerURL, err)
```
Comment (martinmaly) [inline broker-cli/cmd/bindings.go]: Please fix the argument mismatch error (4 placeholders, 5 arguments)

**Comment by deepika-chander** [comment]
Made change fixing go vet + spacing.

**Pull request merged**